thirdeye/thirdeye-frontend/app/pods/manage/alerts/index/route.js

```diff
@@ -1,6 +1,7 @@
 import { hash } from 'rsvp';
 import Route from '@ember/routing/route';
 import fetch from 'fetch';
+import RSVP from 'rsvp';
```

Comment:
are we actually using rsvp here?

Reply:
Let me remove. I had it for testing a promise reject for the error. Thanks.

Reply:
DONE